### PR TITLE
Adding `sy` command

### LINKED ISSUE #9: "sy" macro execution command is missing
### Description

The 'sy' command to execute a macro should be implemented.

The command syntax is:
"""
sy somemacro.sh [args...]
"""

This should execute the script named "/var/www/html/macros/somemacro.sh <args>"
as the correct user (e.g. www-data).

### Additional Information (Optional)

Use cases:

In the Rpi_Cam_Web_Interface the 'sy' command is often used to customize both user buttons and scheduler actions.

For example, I have a script "mktimelapse.sh" that will combine any timelapse filesets into mp4 video files, delete the .jpg files, upload the mp4 to my ftp server, then delete the mp4 filel.
I configure a scheduled action as: "sy mktimelapse.sh 3" in Rpi_Cam_Web_Interface to run this every night.

Something like this could also be used to upload motion-detect videos each night, too.

This kind of thing is used by users for all kinds of things to customize Rpi_Cam_Web_Interface.
It is one of the best features of Rpi_Cam_Web_Interface, I think...

### Checks

- [X] I have not found any related feature requests
- [X] I have not found this feature in the application
- [X] #10

### PR DESCRIPTION
**Pull request by kaihokori**
# Description

## Summary of Changes

- Adding the `sy` command to execute a macro as www-data

## Additional Comments

This PR resolves feature request #9

### CODE REVIEW

**Comment by hogwell** [comment]
This looks great!

A couple things I noticed:

1. As-is, the code is hard coded for the "/var/www/html/macros" path for the script.
However, the original code gets this path from the "macros_path" config option.

2. The original code seems to just run the macro as the current user, rather than switching to www-data explicitly. I now see that this works because raspycam is launched at boot time running as user www-data in rc.local.

Thanks!

File: app/core/process.py [hunk 1]
```diff
@@ -726,3 +735,36 @@ def start_background_process(config_filepath):
...
+    # Check if the script exists and is executable
+    if not os.path.isfile(script_path):
+        print(f"ERROR: Script {script_path} does not exist.")
+        return False
+    if not os.access(script_path, os.X_OK):
+        print(f"ERROR: Script {script_path} is not executable.")
+        return False
+
+    # Construct the command with 'www-data' user context
+    command = ["sudo", "-u", "www-data", script_path] + args
```
Comment (kaihokori) [inline app/core/process.py]: Sorry, I'm a bit tired. Would you mind rewriting that second point? Would you like me to not explicitly state the user context? If this is the case this line would be `command = [script_path] + args`.

Comment (kaihokori) [reply app/core/process.py]: Also, no clue if I said this before. I'm coding blind here as I don't have access to a Pi for a bit. Sorry xD

Comment (hogwell) [reply app/core/process.py]: Yes, that's right. The code does not need to specify a user context.
(My feature request was a little misleading there...) :-)

I'll try to test your code when I get a chance!

Thanks again!

Comment (kaihokori) [reply app/core/process.py]: Ah, no worries. Thanks for clarifying! 

I’ll add that change now. 

Take your time with testing 🙌

**Review by NhanDotJS — APPROVED** [review]

**Pull request merged**